frontend/public/components/routes.tsx

```diff
@@ -353,69 +324,82 @@ const RouteTargetRow: React.FC<RouteTargetRowProps> = ({ route, target }) => (
+              (CNAME) record. Your CNAME record should point to your custom domain{' '}
+              <strong>{{ host }}</strong>, to the OpenShift canonical router hostname,{' '}
+              <strong>{{ routerCanonicalHostname }}</strong>, as the alias.
+            </Trans>
+          </p>
+        </div>
+      }
+    >
+      <Button className="pf-m-link--align-left" type="button" variant="link">
+        <QuestionCircleIcon /> {t('network-route~Do you need to set up custom DNS?')}
+      </Button>
+    </Popover>
+  );
+};
 
-const RouteIngressStatus: React.FC<RouteIngressStatusProps> = ({ route }) => (
-  <>
-    {_.map(route.status.ingress, (ingress: RouteIngress) => (
-      <div key={ingress.routerName} className="co-m-route-ingress-status">
-        <SectionHeading text={`Router: ${ingress.routerName}`} />
-        <dl>
-          <DetailsItem label="Host" obj={route} path="status.ingress.host">
-            {ingress.host}
-          </DetailsItem>
-          <DetailsItem label="Wildcard Policy" obj={route} path="status.ingress.wildcardPolicy">
-            {ingress.wildcardPolicy}
-          </DetailsItem>
-          <DetailsItem
-            label="Router Canonical Hostname"
-            obj={route}
-            path="status.ingress.routerCanonicalHostname"
-          >
-            <div>{ingress.routerCanonicalHostname || '-'}</div>
-            {showCustomRouteHelp(ingress, route.metadata.annotations) && (
-              <CustomRouteHelp
-                host={ingress.host}
-                routerCanonicalHostname={ingress.routerCanonicalHostname}
-              />
-            )}
-          </DetailsItem>
-        </dl>
-        <h3 className="co-section-heading-secondary">Conditions</h3>
-        <Conditions conditions={ingress.conditions} />
-      </div>
-    ))}
-  </>
-);
+const RouteIngressStatus: React.FC<RouteIngressStatusProps> = ({ route }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      {_.map(route.status.ingress, (ingress: RouteIngress) => (
+        <div key={ingress.routerName} className="co-m-route-ingress-status">
+          <SectionHeading text={`${t('network-route~Router:')} ${ingress.routerName}`} />
+          <dl>
+            <DetailsItem label={t('network-route~Host:')} obj={route} path="status.ingress.host">
+              {ingress.host}
+            </DetailsItem>
+            <DetailsItem
+              label={t('network-route~Wildcard policy:')}
+              obj={route}
+              path="status.ingress.wildcardPolicy"
+            >
+              {ingress.wildcardPolicy}
+            </DetailsItem>
+            <DetailsItem
+              label={t('network-route~Router canonical hostname:')}
+              obj={route}
+              path="status.ingress.routerCanonicalHostname"
+            >
+              <div>{ingress.routerCanonicalHostname || '-'}</div>
+              {true && (
```

Comment:
Don't forget to revert this change for testing :)

Reply:
yikes!   -thanks :-)